ui/apps/platform/src/Containers/Policies/PolicyManagementPage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Redirect, Route, Switch } from 'react-router-dom';
+
+import { policiesPath, policyManagementBasePath, deprecatedPoliciesBasePath } from 'routePaths';
+import PoliciesPage from './PoliciesPage';
+
+function PolicyManagementPage() {
+    return (
+        <Switch>
+            <Redirect exact from={policyManagementBasePath} to={policiesPath} />
+            <Redirect exact from={deprecatedPoliciesBasePath} to={policiesPath} />
+            <Route path={policiesPath}>
+                <PoliciesPage />
+            </Route>
```

Comment:
I wonder if there is a way to have existing policy detail URLs redirect to the new nested path.

For instance,
https://staging.demo.stackrox.com/main/policies/a3eb6dbe-e9ca-451a-919b-216cf7ee11f5
could redirect to
https://staging.demo.stackrox.com/main/policy-management/policies/a3eb6dbe-e9ca-451a-919b-216cf7ee11f5

Reply:
i tested this and it works as you described with the existing redirects! :)